frappe/templates/pages/print.py

```diff
@@ -31,14 +28,30 @@ def get_context(context):
 	doc = frappe.get_doc(frappe.form_dict.doctype, frappe.form_dict.name)
 	meta = frappe.get_meta(doc.doctype)
 
+	print_format = get_print_format_doc()
+
 	return {
-		"body": get_html(doc, print_format = frappe.form_dict.format,
-			meta=meta, trigger_print = frappe.form_dict.trigger_print, no_letterhead=frappe.form_dict.no_letterhead),
-		"css": get_print_style(frappe.form_dict.style, frappe.form_dict.format),
+		"body": get_html(doc, print_format = print_format,
+			meta=meta, trigger_print = frappe.form_dict.trigger_print,
+			no_letterhead=frappe.form_dict.no_letterhead),
+		"css": get_print_style(frappe.form_dict.style, print_format),
 		"comment": frappe.session.user,
 		"title": doc.get(meta.title_field) if meta.title_field else doc.name
 	}
 
+def get_print_format_doc(print_format_name=None):
+	"""Returns print format document"""
+	if not print_format_name:
+		print_format_name = frappe.form_dict.format or "Standard"
```

Comment:
There was a case when frappe.form_dict.format turns out to be `templates/print_format/standard.html`. Is this case removed or handled?